### PR TITLE
Rename packages

### PR DESCRIPTION
**Pull request by paul-sachs**
Renaming `connect-query` to `connect-react-query` and `protoc-gen-connect-query` to `protoc-gen-connect-react-query`. This makes it clear that these packages still very much depend on running in a react environment.

These changes will be published as v1

### CODE REVIEW

File: examples/react/basic/package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@connectrpc/connect-query-example-react-basic",
+  "name": "@connectrpc/connect-react-query-example-react-basic",
```
Comment (timostamm) [inline examples/react/basic/package.json:2]:
```suggestion
  "name": "@connectrpc/connect-react-query-example-basic",
```

File: packages/protoc-gen-connect-react-query/package.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@connectrpc/protoc-gen-connect-query",
+  "name": "@connectrpc/protoc-gen-connect-react-query",
```
Comment (timostamm) [inline packages/protoc-gen-connect-react-query/package.json:2]: Shouldn't we rename the npm binary as well?

**Review by timostamm — COMMENTED** [review]
I think we're missing a couple of bits, see comments below. Should the extension for generated files remain `_connectquery.ts`?

File: packages/protoc-gen-connect-react-query/package.json (hunk 2 above)
Comment (paul-sachs) [reply packages/protoc-gen-connect-react-query/package.json:2]: Ah right, yeah, good catch.

**Comment by paul-sachs** [comment]
> Should the extension for generated files remain `_connectquery.ts`?

That's a good question. I think it stands to reason that it should be `_connectreactquery.ts`, even it makes me hurt a little inside.

**Comment by paul-sachs** [comment]
@timostamm how do you feel about using `_cq_react.ts` as a file ending instead of `_connectreactquery.ts` postfixes? I kinda like keeping the postfix short since we're already prefixed with the service and the method.

**Comment by paul-sachs** [comment]
Closing this until we can discuss what renaming would cost us.

**Pull request closed without merging**